contrib/buildsystems/CMakeLists.txt

```diff
@@ -72,10 +72,7 @@ if(USE_VCPKG)
 	message("WIN32: ${WIN32}") # show its underlying text values
 	message("VCPKG_DIR: ${VCPKG_DIR}")
 	message("VCPKG_ARCH: ${VCPKG_ARCH}") # maybe unset
-	message("MSVC: ${MSVC}")
```

Comment:
Would it make sense to _move_, rather than _drop_, these lines?

Reply:
Done, also changed all of them to much nicer `STATUS` messages.